### PR TITLE
Added seralizer for SimulationParam

### PR DESCRIPTION
**Pull request by benflexcompute**

### CODE REVIEW

File: flow360/component/simulation/unit_system.py [hunk 1]
```diff
@@ -35,6 +35,9 @@
 )
 u.dimensions.inverse_area = 1 / u.dimensions.area
 u.dimensions.inverse_length = 1 / u.dimensions.length
+u.dimensions.mass_flux = u.dimensions.mass / u.dimensions.time
```
Comment (yifan-flex) [inline flow360/component/simulation/unit_system.py:38]: We shouldn't call this flux, flux is usually per area. mass_flow_rate?

File: flow360/component/simulation/unit_system.py [hunk 2]
```diff
@@ -35,6 +35,9 @@
...
+u.dimensions.velocity_squared = u.dimensions.length**2 * u.dimensions.time ** (-2)
```
Comment (yifan-flex) [inline flow360/component/simulation/unit_system.py:39]: This is energy per mass. specific_energy?

File: flow360/component/simulation/unit_system.py (hunk 1 above)
Comment (benflexcompute) [reply flow360/component/simulation/unit_system.py:38]: Done

File: flow360/component/simulation/unit_system.py (hunk 2 above)
Comment (benflexcompute) [reply flow360/component/simulation/unit_system.py:39]: Done

File: tests/simulation_params/test_simulation_params.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+                refinements=[UniformRefinement(entities=[my_box], spacing=0.1 * u.m)],
+            ),
+            reference_geometry=ReferenceGeometry(
+                moment_center=(1, 2, 3)  # , moment_length=1.0 * u.m, area=1.0 * u.cm**2
+            ),
+            operating_condition=GenericReferenceCondition(
+                velocity_magnitude=234,
+                thermal_state=ThermalState(temperature=300 * u.K, density=1 * u.g / u.cm**3),
+            ),
+            models=[Fluid()],
```
Comment (yifan-flex) [inline tests/simulation_params/test_simulation_params.py:45]: Can we add some boundary conditions here? Just to make sure that surface_models is working

**Review by yifan-flex — APPROVED** [review]

File: tests/simulation_params/test_simulation_params.py (hunk 3 above)
Comment (benflexcompute) [reply tests/simulation_params/test_simulation_params.py]: Done
